third_party/silabs/efr32_board.gni

```diff
@@ -75,8 +75,11 @@ if (efr32_board == "BRD4304A") {
 } else if (efr32_board == "BRD2601B") {
   efr32_family = "efr32mg24"
   efr32_mcu = "EFR32MG24B310F1536IM48"
+} else if (efr32_board == "BRD2703A") {
```

Comment:
didn't you have to add BRD4162A in here too

Reply:
done